numba/cpython/slicing.py

```diff
@@ -233,3 +233,60 @@ def slice_indices(context, builder, sig, args):
         sig.return_type,
         (sli.start, sli.stop, sli.step)
     )
+
+
+def make_slice_from_constant(context, builder, ty, pyval):
+    sli = context.make_helper(builder, ty)
+    lty = context.get_value_type(types.intp)
+
+    default_start_pos, default_start_neg, default_stop_pos, default_stop_neg, default_step = \
+        [context.get_constant(types.intp, x) for x in get_defaults(context)]
```

Comment:
Please line wrap to 80 chars.

Reply:
FYI: This is copied from earlier in the file.

Reply:
I see this is done, thanks.